languages/lang-ja/Naming.ja.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<naming Lang="ja">
+<naming Lang="ja" Load="Merge">
```

Comment:
If we trigger `Load="Merge"` on the base level here, it will merge into whatever is defined in the base file